components/menu/index.css

```diff
@@ -545,8 +574,6 @@
 .spectrum-Menu-chevron {
 	grid-area: chevronArea;
 	align-self: center;
-	block-size: var(--spectrum-menu-item-checkmark-height);
-	inline-size: var(--spectrum-menu-item-checkmark-width);
 }
```

Comment:
So the `block-size` and `inline-size` were actually styles for the icon (`spectrum-Icon`). which is why the grid area and align-self can be left separately?